app/test_main.py

```diff
@@ -1,3 +1,26 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_add_zeros_when_age_less_then_15() -> None:
+    goals = get_human_age(14, 14)
+    assert goals == [0, 0]
+
+
+def test_should_add_units_when_age_between_15_and_23() -> None:
+    goals = get_human_age(15, 23)
+    assert goals == [1, 1]
+
+
+def test_should_add_deuces_when_age_less_then_28() -> None:
+    goals = get_human_age(27, 27)
+    assert goals == [2, 2]
+
+
+def test_when_age_more_then_28() -> None:
```

Comment:
The test name `test_when_age_more_then_28` could be more descriptive. Consider renaming it to `test_cat_and_dog_age_28`.